src/components/ProductLink/styles.module.scss

```diff
@@ -0,0 +1,51 @@
+.productWrapper {
+  width: 200px;
+}
+.productWrapper:hover {
+  opacity: 1;
+}
+.unavailable {
+  opacity: 0.5;
+}
+.priceContainer {
+  display: flex;
+  gap: 0.25rem;
+}
+.salePrice {
+  color: red;
+}
+
+.prevPrice {
+  text-decoration: line-through;
+}
+
+.imgWrapper {
+  background-color: #dddddd;
+  position: relative;
+  width: 100%;
+  aspect-ratio: 3/4;
+}
+
+.productImg,
+.revealImg {
+  width: 100%;
+  position: absolute;
+  aspect-ratio: 3/4;
+}
+
+.revealImg {
+  display: none;
+}
+
+.imgWrapper:hover .revealImg {
```

Comment:
Reveal image hover effect should happen when user hovers over any part of the component (labels/prices included)
```
.productWrapper:hover .revealImg {
  display: block;
}
```